### PR TITLE
fix(*): Improve demo section with some verbose examples

### LINKED ISSUE #5: Adding to the demos section
- I think the demos section can be expanded into their commands and description. People tend to understand more if the commands are on screen with a "small different than Wikipedia" definition.  
- I have made a similar presentation with a detailed command list which a person can refer to order-wise, helping the viewers understand one by one.  
  
Referring to link - https://docs.google.com/presentation/d/1dLshLydrBL8t_X3nvOodi-LkH8iORocy8uadGrlslGc/edit?usp=sharing

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes #5 
- Made changes to the git Workflow section 
- Made changes to the Demo Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
...
                 <section>
                 <span>
-                  Push Changes to Remote Repo
-                  <pre><code class="hljs" data-trim>git push {remote_name} {repo_name}</code></pre>
+                  Defining Remotes
+                  <pre><code class="hljs" data-trim>git remote -v</code></pre>
+                  <pre><code class="hljs" data-trim>git remote add {remote_name} {repo_url}</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment">Remote is just a fancy way to say where the code is stored on GitHub, and where do you want to push/pull the changes from.</li>
```
Comment (CuriousLearner) [inline index.html]: This is incorrect. Can you explain it in better terms?

File: index.html [hunk 2]
```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
...
-                    <li class="fragment">Push</li>
-                    <li class="fragment">Pull Requests</li>
+                    <li class="fragment">Pushing to a repository requires authentication that needs to be provided in the form of your Github username and password.</li>
                 </ul>
-                </p>
+                <p>
+                </section>
+                <section>
+                <span>
+                  Repeat till you get them right ... <br> Then move on to more advanced topics such as branching, rebasing and cherry-picking commits.
```
Comment (CuriousLearner) [inline index.html]: Indent this 2 more spaces

File: index.html [hunk 3]
```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
                     </p>
                 </section>
             </section>
-            <section>
+            <section id="7">
+                <h2>Demo</h2>
+                <p>
+                <ul>
+                    <li class="fragment">Config thy git</li>
```
Comment (CuriousLearner) [inline index.html]: Config `the` git

Fix the typo

File: index.html (hunk 1 above)
Comment (vipulgupta2048) [reply index.html]: I think I should... done.

**Comment by CuriousLearner** [comment]
Ah, I see I had those comments added, but somehow forgot to submit the review.

Done now :)

File: index.html [hunk 4]
```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
                     </p>
                 </section>
             </section>
-            <section>
+            <section id="7">
+                <h2>Demo</h2>
+                <p>
+                <ul>
+                    <li class="fragment">Config the git</li>
```
Comment (CuriousLearner) [inline index.html]: `Configure git`

File: index.html [hunk 5]
```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
...
-                  <pre><code class="hljs" data-trim>git init</code></pre>
-                  <pre><code class="hljs" data-trim>git clone</code></pre>
-                </span>
+                <span>Configuring git
+                  <pre><code class="hljs" data-trim>git config --global user.name "John Doe"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global user.email "johndoe@example.com"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global core.editor "nano"</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment"> This is done for git to identify who is making the commits, for people who will check later on. We recommend to go with nano in the beginning for the editor of choice.</li>
```
Comment (CuriousLearner) [inline index.html]: I think you might want to say:

`This is done for git to identify who is making the commits. For newbies, we recommend going with nano at the beginning as the editor of choice.`

**Review by CuriousLearner — COMMENTED** [review]
Mentioned review inline.

File: index.html (hunk 5 above)
Comment (vipulgupta2048) [reply index.html]: Yours sounds better. Implementing

File: index.html (hunk 4 above)
Comment (vipulgupta2048) [reply index.html]: Done

**Review by CuriousLearner — APPROVED** [review]
Thanks @vipulgupta2048 !

**Pull request merged**